tests/test_demography.py

```diff
@@ -1494,9 +1494,15 @@ def test_two_pop(self):
         # left endpoints.
         ddb = self.two_pop_example()
         steps = np.linspace(0, 400, 1001)
-        rates, P = ddb.coalescence_rate_trajectory(steps=steps, num_samples=[2, 0])
-        rates1, P1 = ddb.coalescence_rate_trajectory(steps=steps, num_samples=[0, 2])
-        rates2, P2 = ddb.coalescence_rate_trajectory(steps=steps, num_samples=[1, 1])
+        rates, P = ddb.coalescence_rate_trajectory(steps=steps, samples={0: 1})
+        rates1, P1 = ddb.coalescence_rate_trajectory(steps=steps, samples={1: 1})
+        rates2, P2 = ddb.coalescence_rate_trajectory(
+            steps=steps,
+            samples=[
+                msprime.SampleSet(1, population=0, ploidy=1),
+                msprime.SampleSet(1, population=1, ploidy=1),
+            ],
+        )
```

Comment:
this is also fine, but `samples={0:1, 1:1}` is equivalent.

Reply:
Doesn't appear to be, I get:
```
            if time >= 0 and time < 100:
                self.assertAlmostEqual(rA, 1 / (2 * 100))
                self.assertAlmostEqual(rB, 1 / (2 * 1000))
>               self.assertAlmostEqual(rC, 0)
E               AssertionError: 0.0009166666666666666 != 0 within 7 places (0.0009166666666666666 difference)
```

Reply:
Oh, of course not. Hmph.